content/cloud/cloud-command-reference.md

```diff
@@ -11,6 +11,10 @@ url = "https://github.com/fermyon/developer/blob/main/content/cloud/cloud-comman
 - [spin cloud deploy](#spin-cloud-deploy)
 - [spin cloud help](#spin-cloud-help)
 - [spin cloud login](#spin-cloud-login)
+- [spin cloud sqlite](#spin-cloud-sqlite)
+- [spin cloud sqlite delete](#spin-cloud-sqlite-delete)
+- [spin cloud sqlite help](#spin-cloud-sqlite-help)
+- [spin cloud sqlite list](#spin-cloud-sqlite-list)
 - [spin cloud variables](#spin-cloud-variables)
```

Comment:
we will have one more command (@rylev  is currently working on it and can give us a thumbs up when its ready for us to copy/paste into dev docs)
- [spin cloud sqlite help](#spin-cloud-sqlite-execute)

Reply:
Thanks @macolso 
I think I can see the new PR that @rylev is about to have approved and merged. It is at https://github.com/fermyon/cloud-plugin/pull/49

My assumption is that once this new PR is merged we can again use the latest version of Canary via this installation command `spin plugin install --url https://github.com/fermyon/cloud-plugin/releases/download/canary/cloud.json` and then do the `spin cloud sqlite --help` to see what new subcommands and their subcommands may be.

The new `execute` command (I think that is the subcommand) can be added to this PR.

Reply:
@mjia8 could try and access this new subcommand on the new release of canary and update this PR.